### PR TITLE
Select lamp feature

### PR DESCRIPTION
**Pull request by gdozot2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,63 +129,177 @@ according to the settings table below:
...
-            
-/>
+ <openlayers-element  options='{ "enableDraw": false, 
+                                        "information": { "duration": 5000, "title": "Signaler un éclairage public", "content": "Sélectionnez un lampadaire défectueux présent dans l’espace public de la ville." },
+                                        "enableGeolocation": true,
+                                        "enableCenterButton": true,
+                                        "enableRotation": true,
+                                        "mode": {
+                                            "type": "select",
+                                            "radius": 40
```
Comment (slecorne) [inline README.md]: why do we need a radius for select?

File: README.md [hunk 2]
```diff
@@ -129,63 +129,177 @@ according to the settings table below:
...
 ```
 
 ### Validation events
 
 For this scenario, there is one events to listen:
 
 - `position-selected`: This event is sent in two cases:
-  - When all the rules are met and the position is available. The position is stored in an array in event.detail and the array contains the x_coordinate and y_coordinate coordinates of the target center.
-    - event.detail example: [x_coordinate, y_coordinate]
+  - When all the rules are met and the position is available. The information is stored in an object in event.detail and this object contains the id of the lamp post and the geometry in WTK format.
```
Comment (slecorne) [inline README.md]:
```suggestion
  - When all the rules are met and the position is available. The information is stored in an object in event.detail and this object contains the id of the object and the geometry in WTK format.
```

File: README.md [hunk 3]
```diff
@@ -129,63 +129,177 @@ according to the settings table below:
...
+
+```
+ <openlayers-element  options='{ "enableDraw": false, 
+                                        "information": { "duration": 5000, "title": "Signaler un banc cassé", "content": "Positionner le centre de la cible à l’emplacement du banc cassé dans l’espace public" },
+                                        "enableGeolocation": true,
+                                        "enableCenterButton": true,
+                                        "enableRotation": true,
+                                        "mode": {
+                                            "type": "target",
+                                            "radius": 40
```
Comment (slecorne) [inline README.md]: same here what is the radius actually since we select a single point?

File: src/utils/control-icon-manager.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
+import { FullScreen, Zoom } from 'ol/control';
+import { useStore } from '../composable/store';
+import SVGCreator from './svg-creator';
+import GeolocationCenter from '../components/geolocation-center';
+import ResetRotationControl from '../components/reset-rotation-control';
+import InformationControl from '../components/information-control';
+
+export default class ControlIconManager {
```
Comment (slecorne) [inline src/utils/control-icon-manager.ts:23]: actually would probably simplify a lot the css if you would put all control icon inside a container control.
This way you can define the display of buttons inside the container with flex if needed and only change the margin-top of the buttonsContainer when info box is displayed
Something like:
```
class ButtonsContainer extends Control {
  /**
   */
public div: HTMLElement;

  constructor() {
    const element = document.createElement('div');
    element.className = 'buttons-control-container';

    super({
      element: element,
    });

    this.div = element;
  }

}

export default class ControlIconManager {
  static setupIcon() {
    const options = useStore().getOptions();
    const map = useStore().getMap()
    map.addControl(new InformationControl());

    const buttonsContainer = new ButtonsContainer();
    map.addControl(buttonsContainer);
    if (options.fullscreen)
      map.addControl(
        new FullScreen({
          label: SVGCreator.fullScreenLabel(),
          labelActive: SVGCreator.fullScreenLabelActive(),
          className: `ol-full-screen`,
          target: buttonsContainer.div
        })
      );
```

File: README.md (hunk 1 above)
Comment (gdozot2) [reply README.md]: I remove this parameter because not used for the moment (in the doc too)

File: README.md (hunk 2 above)
Comment (gdozot2) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (gdozot2) [reply README.md]: Same I previous

File: src/utils/control-icon-manager.ts (hunk 4 above)
Comment (gdozot2) [reply src/utils/control-icon-manager.ts:23]: Done but with a little change. I have two container. One for the left side and one for the right side.

**Review by slecorne — APPROVED** [review]

**Pull request merged**